README.md

```diff
@@ -213,17 +214,92 @@ quarkus.oidc-client.petstore_auth.grant-options.password.password=alice
 quarkus.oidc-client.petstore_auth.client-id=petstore-app
 ```
 
-The configuration suffix `quarkus.oidc-client.petstore_auth` is exclusive for the schema defined in the specification file.
+The configuration suffix `quarkus.oidc-client.petstore_auth` is exclusive for the schema defined in the specification file and the schemaName is sanitized by aplying the rules described above.
 
-For this to work you **must** add [Quarkus OIDC Client Filter Extension](https://quarkus.io/guides/security-openid-connect-client#oidc-client-filter) to your project:
```

Comment:
@wmedvede if it's not mandatory to native compilation, I'd rather keep this optional, so we won't bring unnecessary dependencies to users' projects.

Reply:
this was restored